pytorch_lightning/plugins/precision/precision_plugin.py

```diff
@@ -201,9 +201,6 @@ def clip_grad_by_norm(self, optimizer: Optimizer, clip_val: Union[int, float]) -
         parameters = self.main_params(optimizer)
         torch.nn.utils.clip_grad_norm_(parameters, clip_val)
 
-    def pre_dispatch(self) -> None:
```

Comment:
@reviewers Accelerator refactor and stable version will have a lot of breaking changes. I think customized precision_plugin + overrides pre_dispatch() use case should be very minor. 
I can change this to deprecated if that is preferred